tests/unit/test_venv.py

```diff
@@ -1233,3 +1235,15 @@ def test_path_change(tmpdir, mocksession, newconfig, monkeypatch):
         path = x.env["PATH"]
         assert os.environ["PATH"] in path
         assert path.endswith(str(venv.envconfig.config.toxinidir) + "/bin")
+
+
+@pytest.mark.parametrize("pythonpath", (None, "blah"))
```

Comment:
if we don't add more parameters there's no need for parametrize here; also use mocker.patch from pytest-mock instead imorting unittest.mock